tests/chainer_tests/test_version.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+import packaging.version
+
+import chainer
+
+
+class TestVersion(unittest.TestCase):
+    def test_pep440(self):
+        version = packaging.version.parse(chainer.__version__)
+        self.assertIsInstance(version, packaging.version.Version)
```

Comment:
Chainer has a rule to make each test file runnable.
```suggestion
        self.assertIsInstance(version, packaging.version.Version)


testing.run_module(__name__, __file__)
```